frame/pallet-account/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "pallet-account"
+authors = ["Stake Technologies"]
+edition = "2021"
```

Comment:
Please use workspace inheritance for relevant package info values.

Reply:
Added